alacritty/src/display/mod.rs

```diff
@@ -853,6 +928,55 @@ impl Display {
             rects.push(visual_bell_rect);
         }
 
+        // Handle search and IME positioning.
```

Comment:
```suggestion
        // Handle IME positioning and search.
```

Would just like to make it clear we're not handling search positioning here. Could maybe also change it from just `search` to `search rendering` or `search drawing`.